app/controllers/employee/parcels_controller.rb

```diff
@@ -1,22 +1,34 @@
 class Employee::ParcelsController < EmployeeController
 
   def index
-    @parcels = Parcel.newest_first
+    @pending_parcels = Parcel.pending.newest_first
+    @accepted_parcels = Parcel.accepted.newest_first
+    @rejected_parcels = Parcel.rejected.newest_first
   end
 
-  # def edit
-  #   @parcel = Parcel.find(params[:id])
-  # end
+  def edit
+    @parcel = Parcel.find(params[:id])
+    @operations = @parcel.operations.newest_first
+  end
 
-  # def update
-  #   parcel = Parcel.find(params[:id])
-  #   parcel.update!(parcel_params)
-  #   redirect_to action: "index"
-  # end
+  def update
+    parcel = Parcel.find(params[:id])
+    acceptance_status = parcel_params[:acceptance_status]
+    author = current_user
```

Comment:
Btw - zrób sobie w `EmployeeController` alias `current_employee`, który będzie prowadził do `current_user`. Nie jestem pewien, czy devise nie ma metody która podmieni Ci to we wszystkich metodach.